static/nodejs_server/utils.ts

```diff
@@ -139,9 +148,13 @@ export function getChartUrl(
       if (!paramVal) {
         return;
       }
```

Comment:
So, we're compressing each value independently?  Is there value in compressing stuff like place, enclosed_place_type.

Or how about just compressing the whole thing in one shot and passing `arg=<base64-encoded-stuff>`?

Reply:
ah, good idea, updated